frontend/src/routes/Video.tsx

```diff
@@ -393,7 +393,7 @@ const Metadata: React.FC<MetadataProps> = ({ id, event }) => {
 
     const sharedStyle = {
         padding: "20px 22px",
-        ...showButton && { paddingBottom: 26 },
+        ...showButton && { paddingBottom: 55 },
```

Comment:
Hrm that's a lot of padding, not a fan. But yeah, the text should be selectable, so I guess this is necessary.

Reply:
Mh I don't mind it and like the breathing room. But if you have an alternative suggestion, feel free to open a PR.